### ISSUE TITLE
Unable to run my lamdaSeleniumNode created by following the tutorial

### ISSUE THREAD

**Issue by sayom88** [blackboard/lambda-selenium#58]
Hello Team(@shurrey ,@varju,@vignesh-s,@wesmcouch)
I am following the tutorial https://medium.com/@Moatazeldebsy/ui-testing-using-selenium-webdriver-and-chrome-inside-aws-lambda-77a17ec64862 and the video given here and setup my AWS Lambda accordingly. I am using a free AWS Tier.

When I run, I get the following error as below. Any help will be highly appreciated. Tnx.
![image](https://user-images.githubusercontent.com/16284729/92308334-7c80e900-efba-11ea-9516-ef1b2e4bb0d5.png)

**Comment by sayom88** [comment]
Hello Team,

Any updates please?

**Comment by sayom88** [comment]
Hi @rprusia did u get similar thing please? Hi @tsu-denim  any help on this please.

**Comment by varju** [comment]
This isn't a project I know anything about. Last I'd heard, @wesmcouch ripped all of the Selenium inside Lambda code out of our Learn builds.

**Comment by wesmcouch** [comment]
Yes I wouldn't recommend using Lambda, as recent versions of Chrome are not easily implemented in recent versions of the Lambda environment and we have abandoned it internally and opted for K8s selenium grids attached to ephemeral pods during build time. If you still want to use Selenium in Lambda use the headless version which should be relatively compatible following the same principles
